### PR TITLE
About branch fixes

### PR DESCRIPTION
**Pull request by Torbins**

### CODE REVIEW

File: LvivCompany.Bookstore/LvivCompany.Bookstore.Web/Startup.cs [hunk 1]
```diff
@@ -66,6 +66,13 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
                     template: "{controller=Home}/{action=Index}/{id?}");
             });
 
+            app.UseMvc(routes =>
+            {
+                routes.MapRoute(
+                    name: "About",
+                    template: "{controller=About}/{action=About}");
```
Comment (IUnknownPtr) [inline LvivCompany.Bookstore/LvivCompany.Bookstore.Web/Startup.cs]: Use generic route template for all controllers. Don't hard code values here

File: LvivCompany.Bookstore/LvivCompany.Bookstore.Web/wwwroot/css/SideBarrStyle.css [hunk 2]
```diff
@@ -164,3 +164,7 @@
         display: block;
     }
 }
+
+.MainContent {
```
Comment (IUnknownPtr) [inline LvivCompany.Bookstore/LvivCompany.Bookstore.Web/wwwroot/css/SideBarrStyle.css]: use lower case naming for class names like `main` or `main-content`

**Review by IUnknownPtr — CHANGES_REQUESTED** [review]

File: LvivCompany.Bookstore/LvivCompany.Bookstore.Web/Startup.cs [hunk 3]
```diff
@@ -72,16 +72,8 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
             {
                 routes.MapRoute(
                     name: "default",
-                    template: "{controller=Home}/{action=Index}/{id?}");
-            });
-
-           
-
-            app.Run(async (context) =>
```
Comment (Torbins) [inline LvivCompany.Bookstore/LvivCompany.Bookstore.Web/Startup.cs]: Цей ран напевно спочатку треба було пофіксати окремим комітом, а уже потім видалити

**Comment by Torbins** [comment]
Everything seems to be ok now

File: LvivCompany.Bookstore/LvivCompany.Bookstore.Web/Views/Shared/_Layout.cshtml [hunk 4]
```diff
@@ -47,7 +47,7 @@
                     <li class="container-fluid"><a href="/Home/Index">Home</a></li>
                     <li class="container-fluid"><a href="#">Delivery</a></li>
                     <li class="container-fluid"><a href="#">Help</a></li>
-                    <li class="container-fluid"><a href="/About/About">About us</a></li>
+                    <li class="container-fluid"><a href="@Url.RouteUrl(new { controller = "About", action = "About" })">About us</a></li>
```
Comment (IUnknownPtr) [inline LvivCompany.Bookstore/LvivCompany.Bookstore.Web/Views/Shared/_Layout.cshtml]: Use anchor tag helper instead

File: LvivCompany.Bookstore/LvivCompany.Bookstore.Web/Startup.cs [hunk 5]
```diff
@@ -30,7 +30,7 @@ public Startup(IHostingEnvironment env)
             Configuration = builder.Build();
         }
 
-        public IConfiguration Configuration { get; }
+        //public IConfiguration Configuration { get; }
```
Comment (IUnknownPtr) [inline LvivCompany.Bookstore/LvivCompany.Bookstore.Web/Startup.cs]: Keep your code clean. Remove code if it's not required, don't comment it. Use comments only for documentation purposes.

**Review by IUnknownPtr — CHANGES_REQUESTED** [review]

File: LvivCompany.Bookstore/LvivCompany.Bookstore.Web/Views/Shared/_Layout.cshtml (hunk 4 above)
Comment (Torbins) [reply LvivCompany.Bookstore/LvivCompany.Bookstore.Web/Views/Shared/_Layout.cshtml]: Fixed

File: LvivCompany.Bookstore/LvivCompany.Bookstore.Web/Startup.cs (hunk 5 above)
Comment (Torbins) [reply LvivCompany.Bookstore/LvivCompany.Bookstore.Web/Startup.cs]: Done.

**Review by IUnknownPtr — CHANGES_REQUESTED** [review]
Please resolve conflicts

**Comment by Torbins** [comment]
Conflicts resolved

**Pull request merged**